diff --git a/private/erlang_build.bzl b/private/erlang_build.bzl
index 64e5a95..39092dc 100644
--- a/private/erlang_build.bzl
+++ b/private/erlang_build.bzl
@@ -159,6 +159,7 @@ tar --create \\
             post_configure_cmds = post_configure_cmds,
             extra_make_opts = extra_make_opts,
         ),
+        use_default_shell_env = True,
         mnemonic = "OTP",
         progress_message = "Compiling otp from source",
     )
